ts/screens/profile/ProfileMainScreen.tsx

```diff
@@ -545,7 +546,27 @@ const mapStateToProps = (state: GlobalState) => ({
 const mapDispatchToProps = (dispatch: Dispatch) => ({
   // hard-logout
   logout: () => dispatch(logoutRequest({ keepUserData: false })),
-  resetPin: () => dispatch(updatePin()),
+  requestIdentificationAndResetPin: () => {
+    const onSuccess = () => dispatch(updatePin());
+
+    return dispatch(
```

Comment:
Since we no need different parameters from the default one we can pass `undefined` 

```suggestion
    return dispatch(
      identificationRequest(
        true,
        false,
        undefined,
        undefined,
        {
          onSuccess
        },
        shufflePinPadOnPayment
      )
    );
```